external_search.py

```diff
@@ -1103,85 +1159,6 @@ def _match_range(cls, field, operation, value):
 class Query(SearchBase):
     """An attempt to find something in the search index."""
 
-    # When we run a simple query string search, we are matching the
-    # query string against these fields.
-    SIMPLE_QUERY_STRING_FIELDS = [
-        # These fields have been stemmed.
-        'title^4',
-        "series^4",
-        'subtitle^3',
-        'summary^2',
-        "classifications.term^2",
-
-        # These fields only use the standard analyzer and are closer to the
-        # original text.
-        'author^6',
-        'publisher',
-        'imprint'
-    ]
```

Comment:
We no longer rely on Lucene syntax to weight fields differently, and we no longer tell Elasticsearch to do blanket searches across large numbers of fields.